rllib/tuned_examples/dqn/stateless-cartpole-r2d2-fake-gpus.yaml

```diff
@@ -11,8 +11,9 @@ stateless-cartpole-r2d2:
         # R2D2 settings.
         replay_buffer_config:
           type: MultiAgentReplayBuffer
+          storage_unit: sequences
           replay_burn_in: 20
-        zero_init_states: true
+          zero_init_states: true
```

Comment:
Can we make sure, using the old setting (config['zero_init_states']) is still supported and we get a warning?